lib/src/model/item_model.dart

```diff
@@ -0,0 +1,57 @@
+class ItemData {
+  String? calories,
+      carbos,
+      description,
+      fats,
+      headline,
+      id,
+      image,
+      name,
+      proteins,
+      thumb,
+      time;
+  int? difficulty;
```

Comment:
1. Always create an object as immutable. See more [why?](https://medium.com/@yetesfadev/immutable-in-dart-and-flutter-understanding-usage-and-best-practices-742be5fa25ea)
2. Each type should never be nullable if we know that its values are not nullable 
3. Linter make your code cleaner and code faster with auto correct. It's best to enable Linter. For e.g.
- [sort_constructors_first](https://dart.dev/tools/linter-rules/sort_constructors_first)
- [require_trailing_commas](https://dart.dev/tools/linter-rules/sort_constructors_first)